modules/react/avatar/stories/avatar/Avatar.mdx

```diff
@@ -9,7 +9,7 @@ import {Size} from './examples/Size';
 import {Variant} from './examples/Variant';
 import {CustomStyles} from './examples/CustomStyles';
 
-import * as AvatarStories from './Avatar.stories'
+import * as AvatarStories from './Avatar.stories';
```

Comment:
The semicolon needs to be present in order for this line to be stripped out properly in [`build-mdx`](https://github.com/Workday/canvas-kit/blob/master/modules/docs/utils/build-mdx.js#L28).